VideoLocker/src/main/java/org/edx/mobile/view/my_videos/MyRecentVideosFragment.java

```diff
@@ -337,6 +337,22 @@ private void playVideoModel() {
         adapter.setVideoId(this.videoModel.videoId);
     }
 
+    private void destroyPlayer() {
+        if (playerFragment == null) {
+            return;
+        }
+        showDeletePanel(getView());
+        playerFragment.lockOrientation();
+        FragmentManager childManager = getChildFragmentManager();
+        childManager.findFragmentById(R.id.container_player);
+        childManager.beginTransaction().remove(playerFragment).commitAllowingStateLoss();
+        View container = getView().findViewById(R.id.container_player);
+        if (container.getVisibility() != View.GONE) {
```

Comment:
nit-pick: No good reason to check if view is already GONE, as there is no harm in setting it to GONE when it's already GONE.